### DISCUSSION TITLE
One Workstation Two Workers

### DISCUSSION THREAD

**Discussion by Swandar** [dorssel/usbipd-win#1051] [Q&A]
Would this allow two people to work simultaneously on one workstation if using a vm like hyperv or vmware? So two mice+cursor (one on main machine and virtual) and two fully functional keyboards?

**Comment by dorssel** [comment]
Possibly, but your description is rather vague. Can you connect 2 physical USB keyboards (possibly connected to two different physical hosts) to one remote VM? Sure.
